siac/gatewaycmd.go

```diff
@@ -108,7 +110,10 @@ func gatewaylistcmd() {
 		return
 	}
 	fmt.Println(len(info.Peers), "active peers:")
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+	fmt.Fprintln(w, "Address\tVersion")
 	for _, peer := range info.Peers {
-		fmt.Println("\t", peer)
+		fmt.Fprintf(w, "  %v\t%v\n", peer.NetAddress, peer.Version)
 	}
+	w.Flush()
```

Comment:
Siac testing will take a lot of work and I don't want it to be a blocker for this PR. I have a branch for siac testing I've been working on for a bit. I'll finish it up after the next header-broadcast PR.